benchmarks/CMakeLists.txt

```
@@ -52,6 +52,17 @@ blt_add_executable(
   SOURCES debuglog_benchmarks.cpp
   DEPENDS_ON ${benchmark_depends})
 
+if (ENABLE_OPENMP)
+  set (benchmark_depends
+      ${benchmark_depends}
+      openmp)
+
+blt_add_executable(
```

The indentation of `blt_add_executable` should be shifted over to the right to be at the same indendation level as the `set` on line 56 since both statements are within the `if (ENABLE_OPENMP)` block.
```suggestion
  blt_add_executable(
```